ISM43362Interface.cpp

```diff
@@ -424,7 +424,7 @@ int ISM43362Interface::socket_send_nolock(void *handle, const void *data, unsign
 
     if (!_ism.send(socket->id, data, size)) {
         debug_if(_ism_debug, "ISM43362Interface: socket_send ERROR\r\n");
```

Comment:
do we have enough information from WIFI module to distinguish btw WOULD_BLOCK and ERROR

Reply:
It seems that this patch is, in fact, not necessary to make TCP test passed!
Let's keep the code with the error code.